vernac/comFixpoint.ml

```diff
@@ -323,14 +323,14 @@ let do_fixpoint_common (fixl : Vernacexpr.fixpoint_expr list) =
   let (_, _, _, info as fix) = interp_fixpoint ~cofix:false fixl in
   fixl, ntns, fix, List.map compute_possible_guardness_evidences info
 
-let do_fixpoint_interactive ~scope ~poly l : Declare.Proof.t =
+let do_fixpoint_interactive ~scope ~poly ?typing_flags l : Declare.Proof.t =
   let fixl, ntns, fix, possible_indexes = do_fixpoint_common l in
-  let lemma = declare_fixpoint_interactive_generic ~indexes:possible_indexes ~scope ~poly fix ntns in
+  let lemma = declare_fixpoint_interactive_generic ~indexes:possible_indexes ~scope ~poly ?typing_flags fix ntns in
```

Comment:
Shouldn't do_fixpoint_common get the flags too? I guess it doesn't matter for guard checking, but it does for univs.

Reply:
Yup will fix.